config/projects/amp-ad-project.yaml

```diff
@@ -0,0 +1,48 @@
+# (REQUIRED) Step 0: Duplicate this file and rename using the updated stack name (see Step 1)
+
+# (REQUIRED) Step 1: Update the stack name below by replacing "example" with your project name
+stack_name: amp-ad-project
+
+stack_tags:
+  # (REQUIRED) Step 2: Update all three values below
+  Department: NDR
+  Project: amp-ad
+  OwnerEmail: william.poehlman@sagebase.org
+
+parameters:
+
+  S3ReadWriteAccessArns:
+    # (REQUIRED) Step 3: Replace the email below with your '@sagebase.org' address
+    - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/william.poehlman@sagebase.org
+
+    # (Optional) Step 4: Uncomment and update the following line(s) to grant additional users with read/write access
+    - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/tess.thyer@sagebase.org
+    # - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/william.poehlman@sagebase.org
+
+  # (Optional) Step 5: Uncomment and update the following line(s) to grant additional users with read-only access
+  S3ReadOnlyAccessArns:
+  #   - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/thomas.yu@sagebase.org
+    - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/daisy.han@sagebase.org
+
+  # (Optional) Step 6: Uncomment and update the following lines to change the S3 bucket lifecycle configuration
+  # EnableDataLifeCycle: Enabled
+  # LifecycleDataTransition: '30'  # In days
+  # LifecycleDataStorageClass: INTELLIGENT_TIERING
+
+  # (Optional) Step 7: Uncomment and update the following line to change when objects are deleted automatically
+  # LifecycleDataExpiration: '183'  # In days (i.e., 6 months)
```

Comment:
Please note that files will be automatically deleted after six months unless this line is uncommented and the value changed.

Reply:
@BrunoGrandePhD the six month time frame sounds good to me. Thanks for checking!